install_bootstrap

```diff
@@ -285,6 +285,9 @@ while [[ "${run_boardwalk^^}" != 'Y' &&  "${run_boardwalk^^}" != 'N' ]] ; do
         aws_default_region='aws_default_region'
         ask_question "What is your AWS region?" "$AWS_DEFAULT_REGION" "AWS region" $aws_default_region
 
+        azul_s3_aws_region='azul_s3_aws_region'
+        ask_question "What is the AWS region where the BDBag for export to FireCloud will temporarily be stored?" "$AZUL_S3_AWS_REGION" "Azul_S3_Aws_Region" $azul_s3_aws_region
```

Comment:
This is also the same region as something else, right? What else?

Reply:
Just me personally following these instructions, I would have no clue what to put for this. Is there more info that can be added to the question to make it more clear?